cg_openmm/simulation/physical_validation.py

```diff
@@ -0,0 +1,161 @@
+import os
+import physical_validation as pv
+from physical_validation.data.simulation_data import SimulationData
+from physical_validation.data.observable_data import ObservableData
+from physical_validation.data.ensemble_data import EnsembleData
+from physical_validation.data.unit_data import UnitData
+import matplotlib.pyplot as pyplot
+from simtk import unit
+import numpy as np
+from openmmtools.multistate import MultiStateReporter, MultiStateSampler, ReplicaExchangeSampler
+from openmmtools.multistate import ReplicaExchangeAnalyzer
+
+kB = (unit.MOLAR_GAS_CONSTANT_R).in_units_of(unit.kilojoule / (unit.kelvin * unit.mole))

```

Comment:
if temperature list has units, should this be in units as well?

Reply:
Ah, I see that temps has units stripped off later.  Maybe kB should keep units . . . we'll have to think how to best do it. 

I did it a little different in expectations_fraction_contacts to try to keep track of the units a bit better.